build/ov.mk

```diff
@@ -6,6 +6,7 @@ OV_MENU_SOURCES = \
 	$(SRC)/Hardware/CPU.cpp \
 	$(SRC)/Hardware/DisplayDPI.cpp \
 	$(SRC)/Hardware/RotateDisplay.cpp \
+	$(SRC)/Hardware/DisplayGlue.cpp \
```

Comment:
Why is this "necessary"?

Reply:
otherwise I get an error during compiling XCSoar:
|   LINK    output/UNIX/bin/OpenVarioMenu
| src/OV/OpenVarioMenu.cpp:175: error: undefined reference to 'Display::DetectInitialOrientation()'
| src/OV/OpenVarioMenu.cpp:1162: error: undefined reference to 'Display::DetectInitialOrientation()'
| collect2: error: ld returned 1 exit status
| make: *** [build/ov.mk:38: output/UNIX/bin/OpenVarioMenu] Error 1
| ERROR: oe_runmake failed